hendrix/utils/__init__.py

```diff
@@ -7,6 +7,13 @@
 
 HENDRIX_DIR = os.path.dirname(os.path.abspath(__file__))
 
+# Default PID Directory
+PID_DIR = os.path.join('/', 'var', 'tmp', 'hendrix')
+if not os.path.isfile(PID_DIR):
```

Comment:
You meant `os.path.exists`? `os.path.isfile` is False for directories, so it will try to re-create one and fail if it exists

Reply:
:+1: - Thanks for iterating on this with me

Reply:
:heavy_check_mark: